### DISCUSSION TITLE
How to remove the installation wizard on npm install?

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#1226] [General] [answered]
Hello,
I have installed Fomantic-UI in my Angular project using `npm i fomantic-ui`, as described [here](https://fomantic-ui.com/introduction/getting-started.html).

Can you tell me how to prevent the Fomantic configuration wizard from prompting after installation?

Since I use dist folder coming from "node_modules/fomantic-ui" I do not need semantic folder and semantic.json file. So how can I prevent it from being created?

**Comment by y0hami** [comment, marked as answer]
@smartm0use Since your not using theming you could just use [`fomantic-ui-css`](https://github.com/fomantic/Fomantic-UI-CSS) which doesn't have an installer and would only install to your `node_modules` directory.

**Comment by lubber-de** [comment]
Closing because of no more response and supposing you are using the advice from @hammy2899

**Comment by lubber-de** [comment]
Interactive auto install is removed in 2.9.0 by #2272
